RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -118,28 +189,31 @@ public bool Generate(ProcessedArgs args)
                 createdJarFilePath = BuildPlugin(definition, args.OutputDirectory);
             }
 
-            LogSummary(createdJarFilePath, generatedSqaleFile, licenseAcceptancePackages);
+            LogSummary(createdJarFilePath, generatedSqaleFile);
 
             return createdJarFilePath != null;
         }
 
-        private void LogSummary(string createdJarFilePath, string generatedSqaleFile, IEnumerable<IPackage> licenseAcceptancePackages)
+        private void LogSummary(string createdJarFilePath, string generatedSqaleFile)
         {
             if (generatedSqaleFile != null)
             {
-                // Log a messsage about the generated sqale file at the end of the process: if we
+                // Log a message about the generated sqale file at the end of the process: if we
```

Comment:
Those are the sorts of decisions to call out in code review comments as it's not possible to tell from the code if the behaviour is intentional or accidental.

The comment doesn't reflect those decisions (the comment is specifically about making sure the log output is _not_ lost in the noise), so it needs to be updated.